src/beanmachine/ppl/__init__.py

```diff
@@ -3,6 +3,7 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from beanmachine.ppl.diagnostics.tools import viz
```

Comment:
You'll need to add this to `__all__` so that the linter won't complain that this is imported but not use :)

Reply:
thanks, I need to setup pre-commit to point to setup.cfg as that's where the flake8 config exists. Do you know of a tool that will add copyright notices for Python?